core/src/modules/ops/record.ts

```diff
@@ -574,8 +574,6 @@ export namespace RecordOps {
       }
 
       return record;
-    } catch (error) {
-      throw error;
```

Comment:
This is doing nothing

Reply:
Shouldn't you remove the `try {}` as well?

Reply:
There's a `finally` block below that we still want to execute. 

We've got

```ts
try {
 // whatever happens
} finally {
  if (toLock) await releaseLock();
}
```